config.gemspec

```diff
@@ -37,9 +37,14 @@ Donate: \e[34mhttps://opencollective.com/rubyconfig/donate\e[0m\n"
 
   # Default RSpec run will test against latest Rails app
   unless ENV['APPRAISAL_INITIALIZED'] || ENV['GITHUB_ACTIONS']
-    gems_to_install = /gem "(.*?)", "(.*?)"(?!, platform: (?!\[:ruby\]))/
-    File.read(Dir['gemfiles/rails*.gemfile'].sort.last).scan(gems_to_install) do |name, version|
-      s.add_development_dependency name, version
+    gems_to_install = /gem "(.*?)", "(.*?)"(?:, platform: \:(.*))?/
+
+    File.read(Dir['gemfiles/rails*.gemfile'].sort.last).scan(gems_to_install) do |name, version, platform|
+      if platform.nil?
```

Comment:
@neilwilliams this condition does not really make sense, as in both variants same code is executed. Can you have a look again?

Reply:
You're right, that looks silly. I'll take a look.

Reply:
Sure! Don't remember why you added it in the first place? Maybe these changes in this file should be reverted?

Reply:
Hi @pkuczynski , I've pushed up a change to this.

The original reason for changing this section was because the original regex was not picking up any of the gems that had a platform condition on them. So for example, on my mac, with normal ruby, it wasn't installing sqlite for me.

Based on your comment, I've improved the condition to use a || instead of writing the line twice.

Hope that's all you need from me - sorry I've taken so long getting back to this!